packages/subgraph-service/hardhat.config.ts

```diff
@@ -13,7 +13,7 @@ const config: HardhatUserConfig = {
     settings: {
       optimizer: {
         enabled: true,
-        runs: 200,
+        runs: 50,
```

Comment:
This is also set in `graphprotocol/contracts/packages/subgraph-service/foundry.toml`. Not sure if they are meant to be kept in-sync...

Reply:
ah yes, thank you! I changed it back to 200 to get the gas report and forgot to switch it back